R/utilities-mpi-error-handling.R

```diff
@@ -0,0 +1,153 @@
+#' @title checkSimulationLoaded
+#' @description Check that the simulation has been loaded on the core.
+#' @param simulation the simulation object to be checked if loaded successfully
+#' @param simulationFilePath the path to the simulation object to be checked if loaded successfully
+#' @export
+checkSimulationLoaded <- function(simulation, simulationFilePath) {
+  success <- FALSE
+  if (identical(simulation$sourceFile, simulationFilePath)) {
+    success <- TRUE
+  }
+  return(success)
+}
+
+#' @title checkLibraryLoaded
+#' @description Check that a library has been successfully loaded on the core.
+#' @param libraryName the library to be checked if loaded successfully
+#' @export
+checkLibraryLoaded <- function(libraryName) {
```

Comment:
Same here. This can be greatly simplified returning the boolean from the check

Reply:
Done